src/template.rs

```diff
@@ -56,16 +56,8 @@ pub fn create_liquid_object(
 }
 
 fn is_within_cargo_project(project_dir: &Path) -> bool {
-    let mut cwd = PathBuf::from(project_dir);
-    cwd.push("dummy");
-    loop {
-        if cwd.with_file_name("Cargo.toml").exists() {
-            break true;
-        }
-        if !cwd.pop() || !cwd.exists() {
-            break false;
-        }
-    }
+    let cwd = PathBuf::from(project_dir);
+    cwd.ancestors().any(|folder| folder.join("Cargo.toml").exists())
```

Comment:
`cwd.ancestors()` would go up, up until the root folder. 

We need a way to stop at the "current project" boundaries, otherwise this would yield unexpected results.

Reply:
I see your point.

But I can also imagine cases where that would lead to surprises. Anyways let's ship it and wait for complaints :)

Reply:
I even suspect this is exactly what `cargo` itself does in order to find the project root if you do a `cargo build` some random place

Reply:
lol, I actually expected this behaviour of cargo - how else should it resolve for workspaces?